components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -3690,11 +3697,30 @@ subroutine ocn_compute_land_ice_flux_input_fields(layerThickness, normalVelocity
 
       !-----------------------------------------------------------------
 
-      if (( trim(config_land_ice_flux_mode) == "off" ) .or.  &
-          ( trim(config_land_ice_flux_mode) == "pressure_only")) then
-         ! nothing to do here
-         return
-      end if
+      if ( trim(config_land_ice_flux_mode) == "off" ) return
+
+      if ( trim(config_land_ice_draft_mode) == "data" ) then
+
+         landIceDraftForSsh = landIceDraft
+
+      elseif ( trim(config_land_ice_draft_mode) == "pressure-dependent" ) then
+
+#ifdef MPAS_OPENACC
+         !$acc parallel loop present(landIceDraftForSsh, landIcePressure)
+#else
+         !$omp parallel
+         !$omp do schedule(runtime)
+#endif
+         do iCell = 1, nCellsAll
+            landIceDraftForSsh(iCell) = -landIcePressure(iCell) / (rho_floatation * gravity)
```

Comment:
Any reason not to just use `config_land_ice_rho_ocean` instead of storing it in `rho_floatation`?

Reply:
I think `rho_floatation` is a clearer description of what `config_land_ice_rho_ocean` is used for here. But if you don't think it improves code readability we can change it.

Reply:
I would assign it to a local variable here and not a module-level variable if it's just for readability.